src/data_provider/src/packages/packageLinuxRpmParserHelperLegacy.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Wazuh SYSINFO
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * January 28, 2021.
```

Comment:
Hi Team

I think the date is wrong

Regards
Hanes

Reply:
Same as above